x-pack/filebeat/input/gcs/test/mock/data.go

```diff
@@ -113,6 +163,50 @@ var gcs_test_new_objectList = `{
 	]
   }`
 
+var gcs_test_latest_objectList = `{
+	"kind": "storage#objects",
+	"items": [
+	  {
+		"kind": "storage#object",
+		"id": "gcs-test-latest/ata.json/1661343619910503",
+		"selfLink": "https://www.googleapis.com/storage/v1/b/gcs-test-latest/o/ata.json",
+		"mediaLink": "https://content-storage.googleapis.com/download/storage/v1/b/gcs-test-latest/o/ata.json?generation=1661343619910503&alt=media",
+		"name": "ata.json",
+		"bucket": "gcs-test-latest",
+		"generation": "1661343619910503",
+		"metageneration": "1",
+		"contentType": "application/json",
+		"storageClass": "STANDARD",
+		"size": "643",
+		"md5Hash": "UjQX73kQRTHx+UyXZDvVkg==",
+		"crc32c": "ZI5qFw==",
+		"etag": "COeWwqK73/kCEAE=",
+		"timeCreated": "2022-08-24T12:20:19.911Z",
+		"updated": "2022-08-24T12:20:19.911Z",
+		"timeStorageClassUpdated": "2022-08-24T12:20:19.911Z"
+	  },
+	  {
+		"kind": "storage#object",
+		"id": "gcs-test-latest/data_3.json/1661343636712270",
+		"selfLink": "https://www.googleapis.com/storage/v1/b/gcs-test-latest/o/data_3.json",
+		"mediaLink": "https://content-storage.googleapis.com/download/storage/v1/b/gcs-test-latest/o/data_3.json?generation=1661343636712270&alt=media",
+		"name": "data_3.json",
+		"bucket": "gcs-test-latest",
+		"generation": "1661343636712270",
+		"metageneration": "1",
+		"contentType": "application/json",
+		"storageClass": "STANDARD",
+		"size": "434",
+		"md5Hash": "eOXjYygu6k6687Uf3vPtKQ==",
+		"crc32c": "hHW/Qw==",
+		"etag": "CM7Ww6q73/kCEAE=",
+		"timeCreated": "2022-08-24T12:20:36.713Z",
+		"updated": "2022-08-24T12:20:36.713Z",
+		"timeStorageClassUpdated": "2022-08-24T12:20:36.713Z"
+	  }
+	]
+  }`
+
 var Gcs_test_new_object_ata_json = `{
```

Comment:
It doesn't look to me like these are used outside this package so they can be unexported. Same for those below. The final two are unused.

Reply:
These are used in the expected map in the tests outside of the package